beacon-chain/sync/rpc_data_column_sidecars_by_root.go

```diff
@@ -96,7 +83,15 @@ func (s *Service) dataColumnSidecarByRootRPCHandler(ctx context.Context, msg int
 		ticker = time.NewTicker(tickerDelay)
 	}
 
-	log.Debug("Serving data column sidecar by root request")
+	if log.Logger.Level >= logrus.TraceLevel {
+		for root, columns := range requestedColumnsByRoot {
```

Comment:
A log line for every request element seems like a lot. Is there another way to group this into 1 or a few log messages? Maybe you could use the result of `PrettySlice` as a map key to group distinct sets of columns and emit a log for each column set with a list of roots?

Reply:
Fixed in 5e82781e7135ce6c33afb7920915d8b97092ba0c.